tools/mo/unit_tests/mo/back/ie_ir_ver_2/emitter_test.py

```diff
@@ -117,3 +126,143 @@ def test_serialize_old_api_map_result(self):
         self.assertTrue("value=\"0,3,1,2\"" in serialize_res)
         self.assertTrue(serialize_res.startswith("b'<net><rt_info>"))
         self.assertTrue(serialize_res.endswith("</rt_info></net>'"))
+
+
+class TestSerialize(unittest.TestCase):
+    @staticmethod
+    def build_graph_with_gather():
+        nodes = {
+            **shaped_parameter('data', int64_array([3, 3]), {'data_type': np.float32, 'type': Parameter.op}),
+            **shaped_parameter('indices', int64_array([1, 2]), {'data_type': np.float32, 'type': Parameter.op}),
+            **valued_const_with_data('axis', int64_array(1)),
+            **regular_op_with_empty_data('gather', {'op': 'Gather', 'batch_dims': 0, 'infer': Gather.infer,
+                                                    'type': Gather.op}),
+            **result('res'),
+        }
+
+        edges = [
+            *connect('data', '0:gather'),
+            *connect('indices', '1:gather'),
+            *connect('axis', '2:gather'),
+            *connect('gather', 'res'),
+        ]
+
+        graph = build_graph(nodes, edges)
+
+        data_node = Node(graph, 'data')
+        Parameter.update_node_stat(data_node, {})
+        indices_node = Node(graph, 'indices')
+        Parameter.update_node_stat(indices_node, {})
+
+        gather_node = Node(graph, 'gather')
+        Gather.update_node_stat(gather_node, {})
+
+        res_node = Node(graph, 'res')
+        Result.update_node_stat(res_node, {})
```

Comment:
I think these are lines are redundant. Can you remove it please?

Reply:
Unfortunately `update_node_stat(res_node, {})` is necessary here. It's needed to set `ie_fileds`, especially `@ports` `@consts` for serializer

https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/openvino/tools/mo/front/extractor.py#L164-L320

without these fields this nodes do not got into xml. So we need these lines